Neos.ContentRepository.Export/src/Asset/ValueObject/SerializedAsset.php

```diff
@@ -48,13 +48,13 @@ public static function fromAsset(Asset $asset): self
         );
     }
 
-    public static function fromJson(string $json): self
+    public static function fromJsonString(string $jsonString): self
```

Comment:
ah nice i see you adjusted it to be in sync with the other parts thanks :)
I for one am perfectly fine with `toJson()` and `fromJsonString()` ... we discussed this briefly already in https://github.com/neos/neos-development-collection/pull/4868#discussion_r1492400082

Reply:
yep. I'm still not very fond of it – not only because it is asynchronous but because JSON is a string by definition. But consistency > correctness in this case I'd say